raiden/storage/sqlite.py

```diff
@@ -1019,36 +1017,22 @@ def _get_query_with_values(self,
                 state_events
             WHERE
                 json_extract(state_events.data,
-                        '$._type') IN ({}) {} {} {} {}
+                        '$._type') IN ({}) {} {} {}
```

Comment:
Are we sure about this removal? i'm just concern about what could happen if the number of parameters differ

Reply:
It used to have two params, one for target, and another one for initator. Now they are one composed param (using AND operator), an I find it cleaner, as there are situations where we can have an initiator and no target, and viceversa. 

If you look to the place where the string is formated, a param was also removed (actually, I merged two params into one)

Reply:
@JONAF2103 check out line `1035` on this file